docs/src/developer/ci.rst

```diff
@@ -0,0 +1,21 @@
+**********************
+Continuous Integration
+**********************
+.. image:: https://travis-ci.org/epistimio/orion.svg?branch=master
+   :target: https://travis-ci.org/epistimio/orion
+
+.. image:: https://codecov.io/gh/epistimio/orion/branch/master/graphs/badge.svg?branch=master
+   :target: https://codecov.io/gh/epistimio/orion
+
+We use travis-ci_ and codecov_ for continuous integration and tox_ to automate the process at
+the repository level.
+
+When a commit is pushed in a pull request, a call to ``$ tox`` will be made by
```

Comment:
I had difficulty reading this paragraph. Maybe breaking it down in a list would help. I also feel this section could have more information. Like code style and quality runs first on travis, if pass, then unit-tests and functional tests, if pass then packaging. Also mention of what codecov send to PR when all tests pass and how to interpret it.